### ISSUE TITLE
Keyboard nav selection not working

### ISSUE THREAD

**Issue by wlla** [nypl-spacetime/maps-by-decade#42]
In map browse, keyboard nav for selecting a set of maps doesn't work, ie, even if user is using the keyboard nav, they would have to use the mouse click to select a subset of maps to explore. Need to update so spacebar or enter key serves same function as mouse click.
